### ISSUE TITLE
Worker node count to be set to 0

### ISSUE THREAD

**Issue by piyerf5** [mjmenger/terraform-f5xc-aws-vpc-site#3]
Set worker node count to 0 in vpc-site.tf

nodes_per_az = "1" to be changed to "0".

**Comment by mjmenger** [comment]
I think we're should default it to zero but allow it to be adjusted through an input parameter.

**Comment by JeffGiroux** [comment]
testing now...I need this to finish my AWS app VM testing.

```
nodes_per_az = var.numWorkerNodesPerAz > 0 ? var.numWorkerNodesPerAz : 0
```

Will require new input 'numWorkerNodesPerAz' set at parent hcl. Stay tuned...

**Comment by mjmenger** [comment]
I'm late to the game, but i think this would be better handled with
variable defaults and constraints.

**Comment by JeffGiroux** [comment]
closed with PR #5
